src/huggingface_hub/hf_api.py

```diff
@@ -1801,7 +1801,7 @@ def create_repo(
             exist_ok (`bool`, *optional*, defaults to `False`):
                 If `True`, do not raise an error if repo already exists.
             space_sdk (`str`, *optional*):
-                Choice of SDK to use if repo_type is "space". Can be "streamlit", "gradio", or "static".
+                Choice of SDK to use if repo_type is "space". Can be "streamlit", "gradio", "docker", or "static".
```

Comment:
```suggestion
                Choice of SDK to use if repo_type is "space". Can be "gradio", "streamlit", "docker", or "static".
```
just to keep consistency with `SPACES_SDK_TYPES` definition

Reply:
oops, I'm too late. Nevermind :)

Reply:
oops sorry!